.github/workflows/ci.yml

```diff
@@ -1,50 +1,42 @@
-# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
-# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+name: Continuous integration
 
-name: Node.js CI
-
-on:
-    push:
-        branches: [master]
-    pull_request:
-        branches: [master]
+on: 
+  - push
 
 jobs:
-    build:
-        runs-on: ubuntu-latest
-
-        strategy:
-            matrix:
-                node-version: [12.x, 14.x, 16.x]
-                cucumber-version: [7.x]
-
-        steps:
-            - uses: actions/checkout@v2
-            - name: Use Node.js ${{ matrix.node-version }}
-              uses: actions/setup-node@v1
-              with:
-                  node-version: ${{ matrix.node-version }}
-            - run: yarn add -P @cucumber/cucumber@${{ matrix.cucumber-version }}
-            - run: yarn run lint
-            - run: yarn run check-fmt
-            - run: yarn run check-readme
-            - run: yarn run test-cover
-            - run: yarn run test-cli
-            - run: yarn run examples --tags @offline
-            - name: Coveralls Parallel
-              env:
-                  COVERALLS_FLAG_NAME: run-${{ matrix.node-version }}-${{ matrix.cucumber-version }}
```

Comment:
still necessary to correctly name the parallel jobs (see https://coveralls.io/builds/42230996)